libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -313,7 +340,11 @@ void AP_BattMonitor::check_failsafes(void)
 {
     if (hal.util->get_soft_armed()) {
         for (uint8_t i = 0; i < _num_instances; i++) {
-            const BatteryFailsafe type = check_failsafe(i);
+            if (drivers[i] == nullptr) {
+                continue;
+            }
+
+            const BatteryFailsafe type = drivers[i]->update_failsafes();
```

Comment:
Perhaps `check_failsafes` would be more appropriate of a non-null function.

Reply:
I'm inclined to think `update` is actually better since it's actually controlling timers/debouncing. I'd expect a check function to be `const`able